__tests__/app.test.js

```diff
@@ -116,3 +115,58 @@ describe('get articles',()=>{
   });
 
 })
+
+describe('GET /api/articles/:article_id/comments',()=>{
+  test("return 200 and comments by article's id", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({body})=>{
+        expect(body.comments).toHaveLength(2);
+            body.comments.forEach((comment) => {
+              expect(comment).toHaveProperty("comment_id", expect.any(Number)),
+              expect(comment).toHaveProperty("body", expect.any(String)),
+              expect(comment).toHaveProperty("votes", expect.any(Number)),
+              expect(comment).toHaveProperty("author", expect.any(String)),
+              expect(comment).toHaveProperty("created_at", expect.any(String)),
+              expect(comment.article_id).toBe(5)
+            })
+      })
+  });
+  test('return 200 and array of objects which is by default sorted by "created_at" in a DESC order', () => {
+    return request(app)
+    .get("/api/articles/5/comments")
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.comments).toBeSortedBy("created_at", { descending: true });
+    });
+  });
+  test("return 200 and message that there are no comments associated with this article", () => {
+    return request(app)
+    .get("/api/articles/4/comments")
+    .expect(200)          
+    .then(({ body }) => {
+      expect(body.comments).toEqual([
+        {
+          message: 'There are no comments associated with this article'
```

Comment:
My suggestion here is to expect an empty array as it's a happy path. I leave it up to you to refactor it or not.

Reply:
done